File: python_semicolon/assignment/taxrate.py
def taxrate(price):
	tax= 0
	if price < 1000000:
		tax = price * (10/100)

		return tax
	

	elif price >= 100000 and price <= 3000000:
		tax = price * (15/100)

		return tax

	elif price >= 3000000 and price <= 5000000:
		tax = price * (20/100)
		return tax
	

	elif price > 5000000:
		tax = price * (25/100)
		return tax
	







carprice = int(input("Enter the price of your car to check your road tax \n"))
road_tax = taxrate(carprice)
print("your road tax is " , road_tax)





	
		

File: python_semicolon/assignment/java/Taxrateapp/Taxrate.java
public class Taxrate{

	public static double taxcalculator ( int price){

		double tax = 1;

		if(price < 1000000){
			tax = price * 10/100;
				return tax;
			

			}


		 if(price >= 1000000 && price <= 3000000){
				tax = price * 15/100;
				return tax;
		}

				System.out.print(tax);

		 if(price >= 3000000 && price <= 5000000){
	
				tax = price * 20/100;

				return tax;
}

		if(price > 5000000){

			tax = price * 25/100;

				
			}

				return tax;	

			}

	


	

}